plugins/catalog-backend/src/ingestion/processors/CodeOwnersProcessor.ts

```diff
@@ -25,6 +25,7 @@ import { filter, get, head, pipe, reverse } from 'lodash/fp';
 import { CatalogProcessor } from './types';
 
 const ALLOWED_LOCATION_TYPES = [
+  'url',
```

Comment:
So `.${gitProvider}` below will become `/.url`?

We'll need the reverse mapping thing in place for this to work. And then the UrlReader thing will probably need to have a helper function for getRepoRoot or something like that, because it's not entirely clear what the URL will be always to that, right?

Reply:
Ah yep, good catch. So I was considering creating a helper that can discover provider types based on the integrations config. Thinking it isn't something that we necessarily have to be passing around like we do with the `UrlReader`. Just a `IntegrationUtils.fromConfig(config)` type thing. Wold love to be able to avoid that for now though

Reply:
But will things break then, for `url` type locations?

Should we hardcode looking in `.github`, `.gitlab` etc instead, no matter what the type actually was? Costs a few more requests to the reader but will at least work :)

Reply:
Oh I'm too slow, you already added that